src/constants/constantSettings.ts

```diff
@@ -1,6 +1,7 @@
 export const AAVE_WELCOME = 'aaveWelcome.json'
 export const FIRST_OPEN = 'firstOpen3.json'
 export const LOCAL_EXPERIMENT_CONFIG = 'remoteConfigSticky.json'
+export const NOTIFS_DISMISSED = 'notifsDismissed.json' // To track state of shown notifications
```

Comment:
I don't think filenames should be public constants. This allows any random code in the app to read or write files that don't belong to it. Instead, I would like to see these constants as *private* variables in the source that's responsible for cleaning / reading / writing this particular file. So, this should move to NotificationViewActions.ts and remove the `export`.